packaging/ubuntu-14.04/rules

```diff
@@ -54,7 +54,7 @@ endif
 # security of the system.  Discuss a proper approach to this for downstreams
 # if and when they approach us
 ifeq ($(shell dpkg-vendor --query Vendor),Ubuntu)
-    VENDOR_ARGS=--enable-nvidia-ubuntu
+    VENDOR_ARGS=--enable-nvidia-ubuntu --enable-partially-static
```

Comment:
mvo mentioned that 14.04 doesn't have a static seccomp, I think we are ok building all dynamic on 14.04, we don't make a core snap for it